frontend/src/app/app.component.css

```diff
@@ -3,4 +3,16 @@
     margin: auto;
     background: white;
     box-shadow: 1px 0px 11px 0px rgba(0, 0, 0, 0.70);
+}
+
+mat-sidenav-container, mat-sidenav-content, mat-sidenav {
+    height: 100%;
+}
+ 
+mat-sidenav {
+    width: 250px;
```

Comment:
we should avoid using pixels, use em/rem/vw/vh
https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9